src/v3/pages/reports/meeting_attendance/components/total_box/index.styles.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box } from '@mui/material';
+import { styled } from '@mui/system';
+import { Select } from '@components/index';
+
+export const StyledSelect = styled(Select)({
+  flex: 1,
+});
+
+export const StyledBox = styled(Box)(({ laptopView }: { laptopView: boolean }) => ({
+  display: 'flex',
+  gap: '16px',
+  flexDirection: laptopView ? 'row' : 'column',
+}));
+
+export const StyledColorBox = styled(Box)<{ backgroudColor: string }>((props) => ({
```

Comment:
@Logan08691: please adjust this to prevent the Invalid prop to DOM error. Thank you.